src/libs/PolicyUtils.ts

```diff
@@ -90,10 +90,7 @@ function getPolicyBrickRoadIndicatorStatus(policy: OnyxEntry<Policy>, policyMemb
  */
 function shouldShowPolicy(policy: OnyxEntry<Policy>, isOffline: boolean): boolean {
     return (
-        !!policy &&
-        policy?.isPolicyExpenseChatEnabled &&
-        policy?.role === CONST.POLICY.ROLE.ADMIN &&
-        (isOffline || policy?.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE || Object.keys(policy.errors ?? {}).length > 0)
+        !!policy && policy?.isPolicyExpenseChatEnabled && (isOffline || policy?.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE || Object.keys(policy.errors ?? {}).length > 0)
```

Comment:
👋 Hey folks, coming from https://github.com/Expensify/App/issues/44811
This PR has deleted `policy?.role === CONST.POLICY.ROLE.ADMIN` check, which resulted in workspaces being displayed in workspace list even if user has no role in them
We've resolved this by adding a `!!policy?.role` check to filter out all of the workspaces in which the user has no role